docs/openapi.yml

```diff
@@ -0,0 +1,156 @@
+openapi: 3.0.3
+info:
+  title: Fiti Flaguin - OpenAPI 3.0
+  description: |-
+    This API does things
+  termsOfService: http://swagger.io/terms/
+  contact:
+    email: pablo.rb@celtiberian.es
+  license:
+    name: MIT
+    url: https://choosealicense.com/licenses/mit/
+  version: 0.0.1
+externalDocs:
+  description: Find out more about Swagger
+  url: http://swagger.io
+servers:
+  - url: https://localhost:5000/api/v3
+tags:
+  - name: phrases
+    description: Everything about your Phrase
+paths:
+  /phrases:
+    get:
+      tags:
+        - phrases
+      summary: Returns phrases
+      description: Returns a list of phrases
+      operationId: getPhrases
+      responses:
+        '200':
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Phrase'
+      security:
+        - bearerAuth:
+            - read:phrase
+  /phrases/{phrasesId}:
```

Comment:
**issue:** I thoght we talked about doing uuids so the clients can provide the id in the put action.

Reply:
Although PUT method must be done with the identifier in as url param

Reply:
yes but the name is set to order id in the get action or Im missunderstoonding something.

Reply:
Also I remember reeding number. My bad we can resolve this.